Intersect.Editor/Localization/Strings.cs

```diff
@@ -779,35 +779,35 @@ public partial struct AnimationEditor
 
             public static LocalizedString general = @"General";
 
-            public static LocalizedString lowerclone = @"Clone From Previous";
+            public static LocalizedString CloneFromPrevious = @"Clone From Previous";
 
-            public static LocalizedString lowerframe = @"Frame: {00}";
+            public static LocalizedString FrameX = @"Frame: {00}";
 
-            public static LocalizedString lowerframecount = @"Graphic Frame Count:";
+            public static LocalizedString FrameCount = @"Frames";
 
-            public static LocalizedString lowerframeduration = @"Frame Duration (ms):";
+            public static LocalizedString FrameDuration = @"Frame Duration (ms)";
 
-            public static LocalizedString lowerframeoptions = @"Frame Options";
+            public static LocalizedString FrameOptions = @"Frame Options";
 
-            public static LocalizedString lowergraphic = @"Graphic:";
+            public static LocalizedString Graphic = @"Graphic";
 
             public static LocalizedString lowergroup = @"Lower Layer (Below Target) ";
 
-            public static LocalizedString lowerhorizontalframes = @"Graphic Horizontal Frames:";
+            public static LocalizedString HorizontalFrames = @"Horizontal";
 
-            public static LocalizedString lowerloopcount = @"Loop Count:";
+            public static LocalizedString LoopCount = @"Loop Count";
 
-            public static LocalizedString disablelowerrotations = @"Disable Rotations";
+            public static LocalizedString DisableRotations = @"Disable Rotations";
 
-            public static LocalizedString lowerplay = @"Play Lower Animation";
+            public static LocalizedString Playback = @"Playback";
```

Comment:
Do you want the JsonProperty IgnoreNull thing on these guys?

Reply:
I totally forgot about that but I don't think that any of the editor strings have it. I honestly don't care though, I consider the editor only a step above abandonware.